src/main/java/helperbot/parser/Parser.java

```diff
@@ -0,0 +1,42 @@
+package helperbot.parser;
+
+
+import helperbot.command.AddCommand;
+import helperbot.command.CheckCommand;
+import helperbot.command.Command;
+import helperbot.command.DeleteCommand;
+import helperbot.command.ExitCommand;
+import helperbot.command.FindCommand;
+import helperbot.command.ListCommand;
+import helperbot.command.MarkCommand;
+import helperbot.exception.HelperBotCommandException;
+
+/**
+ * Represent a class that parse all the command from user's input to corresponding <code>Command</code>.
+ */
+public class Parser {
+
+    /**
+     * Parse the input from user to a <code>Command</code>.
+     * @param message The input from user.
+     * @return A <code>Command</code> containing the message.
+     * @throws HelperBotCommandException The first word of the input is not a valid command.
+     */
+    public static Command parse(String message) throws HelperBotCommandException {
+        String[] splitMessage = message.split(" ");
+        String command = splitMessage[0].toLowerCase();
+
+         return switch (command) {
+             case "bye" -> new ExitCommand();
```

Comment:
perhaps consider using enums here as string type checks may be weak
```suggestion
             case BYE -> new ExitCommand();
```